WoWsShipBuilder.Web/Components/ConsumableSelector.razor

```diff
@@ -34,7 +34,7 @@
                                     @foreach (var consumable in consumableSlot.ConsumableData)
                                     {
                                         <MudMenuItem Style="padding: 0;" OnClick="@(_ => SwapConsumable(consumableSlot, consumable))">
-                                            <MudTooltip Placement="Placement.Right" Color="Color.Transparent">
+                                            <MudTooltip Placement="Placement.Right" Color="Color.Transparent" Style="z-index: 1000000">
```

Comment:
Use realistic values

Reply:
Can't, they are calculated and there is no info outside of "put a high enough number" to fix the issue. i tried with adding and removing 0, that is the minimum. I'm not going to try all the number between them